src/Tools/dotnet-symbol/Properties/Resources.Designer.cs

```diff
@@ -113,9 +113,10 @@ internal class Resources {
         ///
         ///Options:
         ///  --microsoft-symbol-server                         Add &apos;https://msdl.microsoft.com/download/symbols&apos; symbol server path (default).
-        ///  --internal-server                                 Add &apos;https://symweb&apos; internal symbol server path.
         ///  --server-path &lt;symbol server path&gt;                Add a http server path.
-        ///  --authenticated-server-path &lt;pat&gt; &lt;server path&gt;   Add a http PAT authenticated s [rest of string was truncated]&quot;;.
+        ///  --authenticated-server-path &lt;pat&gt; &lt;server path&gt;   Add a http PAT authenticated server path.
```

Comment:
Are these commens meant to have all this xml encoding?

Reply:
They were there before. I'm not sure why.